var/spack/repos/builtin/packages/mrcpp/package.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Mrcpp(CMakePackage):
+    """The MultiResolution Computation Program Package (MRCPP) is a general purpose
+    numerical mathematics library based on multiresolution analysis and the
+    multiwavelet basis which provide low-scaling algorithms as well as rigorous
+    error control in numerical computations."""
+
+    homepage = "https://mrcpp.readthedocs.io/en/latest/"
+    url = "https://github.com/MRChemSoft/mrcpp/archive/v1.2.0-alpha1.tar.gz"
+
+    maintainers = ["robertodr", "stigrj", "ilfreddy"]
+
+    version("1.2.0-alpha1",
```

Comment:
Alpha releases tend to be buggy... Can you add 1.1.0 and make it `preferred=True`?

Reply:
Comment addressed. I've added all released version of the library, but marked the latest stable as preferred.

Reply:
It looks like you added 1.2.0-alpha1 twice by accident.

Reply:
Good catch. Thanks!